lib/restaurant.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+require 'dotenv'
+require_relative 'menu.rb'
+Dotenv.load
+
+class Restaurant
+  include Menu
+
+  def initialize
+    @name = 'Bella'
+  end
+
+  def take_away
+    send_text_confirmation
+  end
+
+  private
+
+  def estimated_delivery_time
+    delivery = Time.now + 3600
+    delivery.strftime("%H:%M")
+  end
+
+  def send_text_confirmation
+    @client = Twilio::REST::Client.new ENV['ACCOUNT_SID'], ENV['AUTH_TOKEN']
+
+    message = @client.account.messages.create(
+      :from => ENV['TWILIO_NUM'],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)